frontend/apps/reader/modules/readerstyletweak.lua

```diff
@@ -771,6 +771,16 @@ local BOOK_TWEAK_INPUT_HINT = T([[
 
 %2]], _("You can add CSS snippets which will be applied only to this book."), BOOK_TWEAK_SAMPLE_CSS)
 
+local CSS_SUGGESTIONS = {
+    { "-cr-hint: footnote-inpage;", _("When set on a block element containing the target id of a href, this block element will be shown as an in-page footnote.")},
+    { "-cr-hint: non-linear-combining;", _("Can be set on some specific DocFragment (ie. DocFragment[id*=16]) to ignore them in the linear pages flow.")},
```

Comment:
Either docfragments/them or docfragment/it.

Reply:
How would you write it after I tell you DocFragment is a element tag name (`<DocFragment>`, like `<b>`), and that I'd like to keep the plural "some specific b(s).. them" ?
"some specific bs (ie b[id*=16])" ?
Feels strange to add a `s` to that tag name - but it might be safe as there is an example in the `(ie. b[id*=16])` just behind it.